2020/python/generate_clean_files.py

```diff
@@ -11,6 +11,7 @@
 import utils
 
 config_file = 'configurations/clean_file_config.yaml'
+bank_config = 'configurations/bank1_config.yaml'
```

Comment:
Do we want this to be a passed in argument?

Reply:
Yes, ideally. I'll ticket it.